spec/presenters/tslr_claims_csv_spec.rb

```diff
@@ -55,7 +57,9 @@
         eligibility.claim_school_name,
         eligibility.employment_status.humanize,
         eligibility.current_school_name,
-        claim.full_name,
+        claim.first_name,
+        nil,
```

Comment:
Might be a good idea to check the middle name is in the CSV, even though it's optional. It's bitten me in the past